
From: miles@mcspd15.ucom.lsi.nec.co.jp (Miles Bader)

Is this something that should be done in <asm-generic/dma-mapping.h>?


---

 25-akpm/include/asm-v850/dma-mapping.h |   11 +++++++++++
 1 files changed, 11 insertions(+)

diff -puN include/asm-v850/dma-mapping.h~v850-dma-mapping-fix include/asm-v850/dma-mapping.h
--- 25/include/asm-v850/dma-mapping.h~v850-dma-mapping-fix	2004-04-07 19:04:46.706211824 -0700
+++ 25-akpm/include/asm-v850/dma-mapping.h	2004-04-07 19:04:46.710211216 -0700
@@ -1 +1,12 @@
+#ifndef __V850_DMA_MAPPING_H__
+#define __V850_DMA_MAPPING_H__
+
+#include <linux/config.h>
+
+#ifdef CONFIG_PCI
 #include <asm-generic/dma-mapping.h>
+#else
+#include <asm-generic/dma-mapping-broken.h>
+#endif
+
+#endif /* __V850_DMA_MAPPING_H__ */

_
